mesh/utils/validate_data.py

```diff
@@ -34,8 +35,12 @@ def validate_json_and_required_fields(request_body, required_fields):
         data = json.loads(request_body)
     except json.JSONDecodeError:
         raise InvalidJsonFormat()
-        
+
+    return validate_required_fields(data, required_fields)
+
+
+def validate_required_fields(data, required_fields):
```

Comment:
Just a question but why wouldn't you want to validate the json along with the fields? I see you use this method instead of using the json one...